src/main/java/com/BlogSite/TestBlogProject/services/UserServiceImpl.java

```diff
@@ -24,38 +24,36 @@ public List<User> getUsers() {
     public Result<User> getUser(Long id) {
         Result<User> result = new Result<>();
         User user = userRepository.findById(id).orElse(null);
-        if(user == null) {
+        if (user == null)
```

Comment:
use optional.isPresent or optional.ifPresent insted of optional.orElse(null)) == null

Reply:
https://stackoverflow.com/questions/52047479/optional-ispresent-vs-orelsenull